src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -40,6 +41,16 @@
 	 */
 	public static final String KUBERNETES_DEPLOYMENT_NODE_SELECTOR = "spring.cloud.deployer.kubernetes.deployment.nodeSelector";
 
+	private Config fabric8 = new Config();
```

Comment:
Will create a story to add documentation for all the available fabric8 configuration properties once this PR is accepted and merged.